{% set title = title|default('500 - Internal Server Error') %}
{% extends "base.html" %}
{% block metadata %}
<meta property="og:url" content="https://yutify.onrender.com/404">
<meta property="og:type" content="website">
<meta property="og:title" content="yutify: Home">
<meta property="og:description"
    content="The content you're looking for is not found. This is simple RESTful API for retrieving music info for various steaming platforms.">
<meta property="og:image" content="https://yutify.onrender.com/static/music.png">
<meta name="twitter:card" content="summary_large_image">
<meta property="twitter:domain" content="yutify.onrender.com">
<meta property="twitter:url" content="https://yutify.onrender.com/404">
<meta name="twitter:title" content="yutify: Home">
<meta name="twitter:description"
    content="The content you're looking for is not found. This is simple RESTful API for retrieving music info for various steaming platforms.">
<meta name="twitter:image" content="https://yutify.onrender.com/static/music.png">
{% endblock %}

{% block header %}
<h1>{{ title }}</h1>
{% endblock %}

{% block content %}
<main class="container">
<div class="grid error-container">
    <img class="error-img" src="{{ url_for('static', filename='errors/500.gif') }}" alt="anime-satania" width="500px">
    <div>
        <p>Oops~ the server fainted. Why not go back <a href="{{ url_for('main.index') }}">Home</a>?</p>
        <p>If this keep happening, feel free to open an issue on Github
            <a href="https://github.com/CheapNightbot/yutify/issues">HERE</a> !
            Make sure to provide the input that caused this error.
        </p>
    </div>
</div>
</main>
{% endblock %}
